viewer/viewer.js

```diff
@@ -274,7 +275,8 @@ if (Config.get("passwordSecret")) {
     if (internals.userNameHeader !== undefined) {
       if (req.headers[internals.userNameHeader] !== undefined) {
         var userName = req.headers[internals.userNameHeader];
-        Db.getUserCache(userName, function(err, suser) {
+
+        var ucb = function(err, suser) {
```

Comment:
this can just be `function ucb(err, suser)`

Reply:
Done.  I agree that the whole JSON in the ini file sounded better than it looks...  Functional but ugly...